Modix.Services/Starboard/StarboardHandler.cs

```diff
@@ -15,12 +16,14 @@ public class StarboardHandler :
     {
         private readonly StarboardService _service;
         private readonly IDesignatedChannelService _designatedChannelService;
-        private const string _contentString = "**{0}** {1} ID: {2}";
+        private readonly IQuoteService _quoteService;
+        private const string _contentString = "**{0}** {1}";
 
-        public StarboardHandler(StarboardService service, IDesignatedChannelService designatedChannelService)
+        public StarboardHandler(StarboardService service, IDesignatedChannelService designatedChannelService, IQuoteService quoteService)
```

Comment:
This is overflowing common file width. Consider newlining each of these injections.